misk/src/main/kotlin/misk/security/authz/AccessInterceptor.kt

```diff
@@ -86,9 +67,25 @@ class AccessInterceptor private constructor(
           |
           |
           """.trimMargin()
-          )
+      }
+
+      // This action is explicitly marked as unauthenticated.
+      if (action.hasAnnotation<Unauthenticated>()) {
+        check(actionEntries.size == 1) {
+          val otherAnnotations = actionEntries.filterNot { it.annotation == Unauthenticated::class }
```

Comment:
There is no coverage for this case. What I'd like is a way to assert in a test that the injector failed to create with the given error message. Reviewers, is there a way to do this?

Reply:
Done, thanks! I had to create the injector with Stage.PRODUCTION to get the error to surface. I _think_ this is expected behavior.